### PR TITLE
update documentation for K 3.4 release

### PR DESCRIPTION
**Pull request by dwightguth**
@traiansf @grosu please review

### CODE REVIEW

File: INSTALL.md [hunk 1]
```diff
@@ -1,27 +1,29 @@
...
-
-4. (Optional) To use the pdf backend, a relatively recent installation 
-of Latex is required.  Besides basic packages, the following (texlive)
-packages are needed: 
-  bera, bezos, bookmark, datetime, ec, etoolbox, fancybox, fancyvrb, import, 
-  listings, marginnote, microtype, ms, pdfcomment, pgf (up-to-date), preview, 
-  soul, stmaryrd, times, titlesec, ucs, url, xcolor, and xkeyval.
+1. Prerequisites:
+  * Java Runtime Edition version 7 or higher (http://java.com/en/download/index.jsp)
+  * To make sure java is installed properly, call `java` in a terminal.
```
Comment (traiansf) [inline INSTALL.md]: maybe `java -version` ?

File: README.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+website, which contains video tutorials, at http://www.kframework.org/.
+
+**NOTE**: This README file contains information regarding the stable release of
+the K tool indicated in the title above, regardless of whether it came with
+the release itself or with subsequent nightly/latest builds.  This file is
+updated only when new stable versions are officially released.
+
+**WARNING**: The command line options for kompile, krun, kast and ktest have
+recently changed!
+Type `--help` with any of these to see the new options, or see the changelog
```
Comment (grosu) [inline README.md]: `CHANGELOG.md` instead of `CHANGELOG`?

File: README.md [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+for more details.
+
+New features
+------------
+
+For a list of high-level changes since the previous release, please refer to
+the file CHANGELOG in the current directory.
```
Comment (grosu) [inline README.md]: `CHANGELOG.md` instead of `CHANGELOG`?

File: README.md [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+
+Developers
+----------
+
+See src/README
```
Comment (grosu) [inline README.md]: `README.md`?  Maybe it is better to actually only keep the capitalization but not the extension of these files.

**Comment by grosu** [comment]
Except my comments above, LGTM.

**Pull request merged**